tests/load/nwp/test_load_nwp.py

```diff
@@ -102,3 +103,18 @@ def test_load_excarta_local():
         raise ValueError(
             "The following dimensions are missing: %s" % (str(dim_keys - set(metadata.dims)))
         )
+
+
+def test_load_gfs():
+    nwp_datapipe = OpenNWP(
+        zarr_path="tests/data/gfs.zarr",
+        provider="gfs",
+    )
+    metadata = next(iter(nwp_datapipe))
+    assert metadata is not None
+    assert type(metadata) == DataArray
+    dim_keys = set(["channel", "init_time_utc", "latitude", "longitude", "step"])
+    if bool(dim_keys - set(metadata.dims)):
```

Comment:
Definitely a nit, but I think `bool` is unnecessary here in the if statement, although I can see this is done in all tests in this file

Reply:
probably because they are all done by me! and yes good point, will remove